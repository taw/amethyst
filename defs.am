def foo()
    print("Foo called\n")
end

def bar
    puts("Bar called\n")
end

def two_times()
    yield()
    yield()
end

two_times {
    foo()
    bar()
}
