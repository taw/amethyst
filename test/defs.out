Foo called
Bar called

Foo called
Bar called

